server/config/environment.go

```diff
@@ -200,5 +202,14 @@ func GetEnvironment() envVars {
 		environment.SAVE_CONTAINER = saveContainer
 	}
 
+	environment.SEGMENT_WRITE_KEY = env.Get("SEGMENT_WRITE_KEY")
+	if environment.SEGMENT_WRITE_KEY == "" {
+		log.Warnf("SEGMENT_WRITE_KEY environment variable unset, deployment telemetry will not be available on Amplitude")
```

Comment:
This message should not mention Amplitude but instead it should say that telemetry/metrics wont be publised to Segment.
Also, instead of saying that it is "unset" it should say that environment variable is either not set or is empty string. 
Same changes should be made below.